src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/AddTransparentDataEncryptionCertificateTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.Azure.Commands.ScenarioTest.SqlTests;
```

Comment:
@nivimsft please add a license header to this file

Reply:
Addressed